tests/Doctrine/Tests/DBAL/Types/DateImmutableTypeTest.php

```diff
@@ -24,7 +25,7 @@ class DateImmutableTypeTest extends \PHPUnit\Framework\TestCase
     protected function setUp()
     {
         $this->type = Type::getType('date_immutable');
-        $this->platform = $this->prophesize(AbstractPlatform::class);
+        $this->platform = $this->getMockBuilder(AbstractPlatform::class)->getMock();
```

Comment:
Probably could use `$this->createMock()` instead.

Reply:
Yes, forgot to change this file. :smiley: